### PR TITLE
fix(test): integrate each test

### PR DESCRIPTION
**Pull request by hs0225**
Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

**Comment by hs0225** [comment]
Base on #13

**Review by daeyeon — APPROVED** [review]

File: packages/tests/example/test/widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```
Comment (JSUYA) [inline packages/tests/example/test/widget_test.dart:1]: This test file looks to be unused file. You can delete it after check.

File: packages/tests/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ignore_for_file sort_pub_dependencies
```
Comment (JSUYA) [inline packages/tests/pubspec.yaml:1]: I have a question. Do we really need this file?

**Review by JSUYA — COMMENTED** [review]
Do you need the example directory? In flutterfire, tests is the app path.
https://github.com/firebase/flutterfire/tree/master/tests

**Comment by hs0225** [comment]
@JSUYA Yes, our test utility is implemented based on the example folder.

**Comment by JSUYA** [comment]
> @JSUYA Yes, our test utility is implemented based on the example folder.

I mean, the directory is duplicated.
flutterfiire -> {root}/tests/integration_test/{each package's test}/*.dart
our flutterfiire -> {root}/packages/tests/`example`/integration_test/{each package's test}/*.dart

+ How about moving `tests` directory to repo root path?

File: packages/tests/example/test/widget_test.dart (hunk 1 above)
Comment (hs0225) [reply packages/tests/example/test/widget_test.dart]: I removed this file.

**Review by JSUYA — APPROVED** [review]

**Pull request merged**